[rospy.client][INFO] 2018-07-06 11:01:52,675: init_node, name[/rostopic_9298_1530889312657], pid[9298]
[xmlrpc][INFO] 2018-07-06 11:01:52,676: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2018-07-06 11:01:52,676: Started XML-RPC server [http://mdas-Meerkat:42757/]
[rospy.init][INFO] 2018-07-06 11:01:52,676: ROS Slave URI: [http://mdas-Meerkat:42757/]
[rospy.impl.masterslave][INFO] 2018-07-06 11:01:52,676: _ready: http://mdas-Meerkat:42757/
[rospy.registration][INFO] 2018-07-06 11:01:52,676: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2018-07-06 11:01:52,676: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2018-07-06 11:01:52,776: registered with master
[rospy.rosout][INFO] 2018-07-06 11:01:52,777: initializing /rosout core topic
[rospy.rosout][INFO] 2018-07-06 11:01:52,781: connected to core topic /rosout
[rospy.simtime][INFO] 2018-07-06 11:01:52,784: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2018-07-06 11:01:52,843: topic[/fix] adding connection to [http://mdas-Meerkat:34155/], count 0
[rospy.internal][INFO] 2018-07-06 11:01:53,080: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2018-07-06 11:01:58,260: signal_shutdown [signal-2]
[rospy.internal][INFO] 2018-07-06 11:01:58,266: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2018-07-06 11:01:58,267: topic[/fix] removing connection to http://mdas-Meerkat:34155/
[rospy.internal][WARNING] 2018-07-06 11:01:58,270: Unknown error initiating TCP/IP socket to mdas-Meerkat:46437 (http://mdas-Meerkat:34155/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2018-07-06 11:01:58,270: Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/topics.py", line 307, in close
    c.close()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 807, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.impl.masterslave][INFO] 2018-07-06 11:01:58,271: signal-2
[rospy.core][INFO] 2018-07-06 11:01:58,435: signal_shutdown [atexit]
